pkg/manifest/os.go

```diff
@@ -287,13 +305,15 @@ func (p *OS) getPackageSetChain(Distro) []rpmmd.PackageSet {
 			ps := rpmmd.PackageSet{
 				Include:      workloadPackages,
 				Repositories: append(osRepos, p.Workload.GetRepos()...),
+				// Although 'false' is the default value, set it explicitly to make
+				// it visible that we are not adding weak dependencies.
+				InstallWeakDeps: false,
```

Comment:
Is this in fact what we want? I *thought* we want workload packages (which include blueprint packages, right?) to be non-weak? :)

Reply:
The idea is that as long as blueprint users don't have a mechanism for excluding packages, installing weak dependencies gives them no way to trim their blueprint-installed packages down.